app/component/map/generic-marker.cjsx

```diff
@@ -60,9 +60,14 @@ class GenericMarker extends React.Component
   shouldComponentUpdate: (nextProps) ->
     return nextProps.id != @props.id
 
+  openPopup: =>
+    @refs['marker'].getLeafletElement().openPopup()
+
   getMarker: ->
     <Marker map={@props.map}
+            ref="marker"
             position={lat: @props.position.lat, lng: @props.position.lon}
+            interactive={false}
```

Comment:
The `react-leaflet/Marker` component forwards all unhandled props like this to the underlying `leaflet/marker` implementation. We use Leaflet 1.0 beta where `clickable` has been renamed as `interactive`: http://mourner.github.io/Leaflet/reference.html#marker Setting `interactive={false}` stops the marker from handling events so that the map gets to handle them.